src/package-scripts.js

```diff
@@ -1,7 +1,7 @@
 const lintDirs = 'src/ test/';
 module.exports = {
-  'test': 'istanbul cover test/', // eslint-disable-line
-  'postest': 'istanbul check --statements 90 --functions 90 --branches 90', // eslint-disable-line
+  'test': 'nyc node test/', // eslint-disable-line
+  'postest': 'nyc check-coverage --lines 90 --functions 90 --branches 90', // eslint-disable-line
```

Comment:
In general, how do we come up with the threshold (90% in this case)? any standard out there or it's upto the developer/project.